Code/BasicFilters/json/CannyEdgeDetectionImageFilter.json

```diff
@@ -21,8 +21,8 @@
       "type" : "double",
       "default" : "0.0",
       "pixeltype" : "Input",
-      "briefdescriptionSet" : "Set the Threshold value for detected edges.",
-      "detaileddescriptionSet" : "TODO: Document in the ITKv4 migration guide that the SetThreshold member function was removed from the CannyEdgeDetectionImageFilter , and that both UpperThreshold and LowerThreshold need to be set. To get the same results as with the SetThreshold method change \"myfilter->SetThrehsold\" to \"myfilter->SetUpperThreshold\", and add \"myfilter->SetLowerThreshold(GetUpperThreshold()/2.0)\"",
```

Comment:
This may have been a manually added comment to SimpleITK to document differ. Please retain the detailed description.